waifus/sagisawa chiho.json

```diff
@@ -0,0 +1,8 @@
+{
+	"type" : 0,
+	"english-name" : "Sagisawa Chiho",
+	"image" : "https://static.zerochan.net/Sagisawa.Chiho.full.627144.jpg",
+	"japanese-name" : "鷺澤 千帆",
+	"age-in-show": 16,
```

Comment:
Can you give a source to where this information came from? I can't find it in the source you gave

Reply:
Changed it @yourWaifu